pkg/cosign/rego/rego.go

```diff
@@ -33,23 +34,31 @@ func ValidateJSON(jsonBody []byte, entrypoints []string) error {
 
 	query, err := r.PrepareForEval(ctx)
 	if err != nil {
-		return err
+		errs = append(errs, err)
 	}
 
 	var input interface{}
 	dec := json.NewDecoder(bytes.NewBuffer(jsonBody))
 	dec.UseNumber()
 	if err := dec.Decode(&input); err != nil {
-		return err
+		errs = append(errs, err)
```

Comment:
Should we not `return []error{err}` here? It's unlikely that the rego eval below does anything useful if we feed it bad input.

Reply:
yes, totally agree with you, thank you so much.